ivy-resources/src/main/res/values/strings.xml

```diff
@@ -130,7 +130,7 @@
     <string name="no_transactions_for_period">You don\'t have any transactions for %1$s.\nYou can add one by scrolling down and tapping "Add income" or "Add expense" button at the top.</string>
     <string name="account_confirm_deletion_description">Note: Deleting this account will remove it permanently and delete all associated transactions with it.</string>
     <string name="category_confirm_deletion_description">Note: Deleting this category will remove it permanently.</string>
-    <string name="account_confirm_deletion_type_account_name">Please type \"%s\" in order to delete your account.</string>
+    <string name="account_confirm_deletion_type_account_name">Please type /"cash"/ in order to delete your account.</string>
```

Comment:
Hey @Ashurajput02 the user needs to type the name of the account. Not always "Cash". Logic:
- enable the delete button only if the name of the account is written correctly

Reply:
that is not an issue we can easily replace it with the account name instead of "cash"

Reply:
@ILIYANGERMANOV 
after making these changes , when in the beginning, the user was not even able to write , he is able to write but the cursor is not moving forward as he writes,, can you please figure out if i have done some mistakes in taking the input or is it some flaw in the UI itself so that i can see the code of UI